config/settings/production.yml

```diff
@@ -46,6 +46,7 @@ features:
     sync_trn_data: true
   dqt_import:
     sync_teachers: true
+  change_accredited_provider: true
```

Comment:
we can just remove this and other instances of it since `settings.yml` will take precedence (once we remove the others)